touch-layouts/layouts/forza.json

```diff
@@ -1,244 +1,524 @@
+                      "value": "dPad"
+                    }
+                  }
+                }
+              },
+              null
+            ],
+            [
+              {
+                "type": "button",
+                "action": "leftThumb"
+              },
+              null
+            ]
+          ]
+        },
+        "right": {
+          "inner": [
+            {
+              "type": "button",
+              "action": "rightTrigger",
+              "pullAction": "gamepadA",
+              "styles": {
+                "default": {
+                  "faceImage": {
+                    "type": "icon",
+                    "value": "gasPedal"
+                  }
+                }
+              }
+            }
+          ],
+          "outer": [
+            [
+              {
+                "type": "button",
+                "action": "gamepadB",
+                "styles": {
+                  "default": {
+                    "faceImage": {
+                      "type": "icon",
+                      "value": "add"
+                    }
+                  }
+                }
+              },
+              null
+            ],
+            {
+              "type": "button",
+              "action": "gamepadA",
+              "styles": {
+                "default": {
+                  "faceImage": {
+                    "type": "icon",
+                    "value": "handbrake2"
+                  }
+                }
+              }
+            },
+            [
+              {
+                "type": "button",
+                "action": "leftTrigger",
+                "styles": {
+                  "default": {
+                    "faceImage": {
+                      "type": "icon",
+                      "value": "brakePedal"
+                    }
+                  }
+                }
+              },
+              null
+            ],
+            {
+              "type": "button",
+              "action": "gamepadX",
+              "styles": {
+                "default": {
+                  "faceImage": {
+                    "type": "icon",
+                    "value": "subtract"
+                  }
+                }
+              }
+            },
+            [
+              {
+                "type": "button",
+                "action": "gamepadY",
+                "styles": {
+                  "default": {
+                    "faceImage": {
+                      "type": "icon",
+                      "value": "repeatRefresh"
+                    }
+                  }
+                }
+              },
+              null
+            ]
+          ]
+        },
+        "upper": {
+          "right": [
+            {
+              "type": "button",
+              "action": "menu"
+            },
+            {
+              "type": "button",
+              "action": "view"
+            }
+          ]
+        }
+      }
+    },
+    "layout-2": {

```

Comment:
I'd prefer this to be called `layout-1-throttle-pedal` and `Layout 1 (Throttle Pedal)`, since it's still Layout 1

Reply:
OK. you can change your name if you want.